### PR TITLE
Added docs for Yaml::PARSE_CONSTANT

### LINKED ISSUE #7178: Add docs for YAML constants
In https://github.com/symfony/symfony/pull/18626 we added a new `Yaml::PARSE_CONSTANT` flag to support PHP constants in PHP files.

### PR DESCRIPTION
**Pull request by javiereguiluz**
This fixes #7178.

### CODE REVIEW

**Comment by chalasr** [comment]
👍 

Status: reviewed

File: components/yaml.rst [hunk 1]
```diff
@@ -293,6 +293,17 @@ flag::
     //       Line
     //       String
 
+Parsing PHP Constants
+~~~~~~~~~~~~~~~~~~~~~
+
+By default the YAML parser treats the PHP constants included in the contents as
```
Comment (xabbuh) [inline components/yaml.rst]: comma after "default"?

**Comment by xabbuh** [comment]
👍

**Comment by HeahDude** [comment]
Thanks again :)

**Comment by xabbuh** [comment]
Thank you Javier.

**Pull request closed without merging**